typings/index.d.ts

```diff
@@ -282,6 +282,8 @@ export class Command {
   readonly commands: readonly Command[];
   readonly options: readonly Option[];
   readonly registeredArguments: readonly Argument[];
+  /** @deprecated */
+  readonly _args: readonly Argument[]; // added here for strikethrough highlighting in editors
```

Comment:
When is this better than leaving it undeclared, and hence an error to use?

Reply:
When coding in JavaScript and `checkJs` is not turned on in the TSConfig.

Reply:
Feels slightly backwards, but it is typical to add warnings about deprecated things before removing them. Ok.